src/app/helpers/worldgen.ts

```diff
@@ -384,6 +384,113 @@ function addCornerNodes(
   });
 }
 
+function fillFogGaps(
+  config: WorldConfigContent,
+  nodes: Record<string, WorldLocation>,
+  rng: PRNG,
+  counts: Record<LocationType, number>,
+): void {
+  // Import revelation radius constants locally to avoid circular imports
+  const REVELATION_RADIUS: Record<LocationType, number> = {
```

Comment:
refactor REVELATION_RADIUS from fog-of-war to add an export in `interfaces/world.ts`. import that in both locations so it isn't copypasted everywhere.